src/hooks/api/useApiVaults/useApiCompounderVaults.ts

```diff
@@ -3,43 +3,44 @@ import { Address } from "wagmi"
 
 import fortressApi, { ApiResult } from "@/lib/fortressApi"
 import { VaultType } from "@/hooks/types"
+import useActiveChainId from "@/hooks/useActiveChainId"
 import useIsCurve from "@/hooks/useIsCurve"
 import useIsTokenCompounder from "@/hooks/useIsTokenCompounder"
 
-import { CHAIN_ID } from "@/constant/env"
+type UseApiCompounderVaultsParams = {
+  type: VaultType
+}
 
-export default function useApiListCompounderVaults({
+export default function useApiCompounderVaults({
```

Comment:
Better to just use two hooks for type safety.

We should always aim to have a single, centralized hook for requesting data with react-query so that it is easy to update in one place than scattered throughout the app. It also will reduce errors when updating the query keys like I did in this PR.